### PR TITLE
add(sepolia): as the main testnet

### PR DESCRIPTION
**Pull request by AlbionaHoti**
## What 💻
Add sepolia sources for the network

## Why ✋
Migrating to Sepolia testnet

### CODE REVIEW

File: contracts/deploy/erc20.ts [hunk 1]
```diff
@@ -20,8 +21,6 @@ if (!RECIPIENT_ADDRESS)
 
 export default async function (hre: HardhatRuntimeEnvironment) {
   console.log(`Running deploy script for the MyERC20 contract...`);
-  const provider = new Provider("https://testnet.era.zksync.dev");
```
Comment (dutterbutter) [inline contracts/deploy/erc20.ts:23]: Shouldn't this be updated rather then removed?

File: contracts/deploy/allowListPaymaster.ts [hunk 2]
```diff
@@ -1,10 +1,12 @@
-import { Provider, Wallet } from "zksync-web3";
 import * as ethers from "ethers";
-import { HardhatRuntimeEnvironment } from "hardhat/types";
-import { Deployer } from "@matterlabs/hardhat-zksync-deploy";
 
+import { Provider, Wallet } from "zksync-web3";
+
+import { Deployer } from "@matterlabs/hardhat-zksync-deploy";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
```
Comment (dutterbutter) [inline contracts/deploy/allowListPaymaster.ts:6]: Any particular reason you've updated the ordering here?

**Review by dutterbutter — CHANGES_REQUESTED** [review]
A couple comments. 

Can you resolve / review the CI issue. And can you confirm you've tested the deployment, and interaction locally?

File: contracts/deploy/allowListPaymaster.ts (hunk 2 above)
Comment (AlbionaHoti) [reply contracts/deploy/allowListPaymaster.ts:6]: My settings on automatically sort imports did this, reverting it back.

**Comment by dutterbutter** [comment]
@AlbionaHoti please address / review test failure.

**Comment by dutterbutter** [comment]
@AlbionaHoti I will fix this CI.

**Review by dutterbutter — APPROVED** [review]

**Pull request merged**